docs/conf.py

```diff
@@ -22,6 +22,8 @@
 
 django.setup()
 
+from django_setup_configuration.documentation.model_directive import setup
```

Comment:
is there a way to add this to `extensions` instead of having to do an import?

Reply:
will look into it!

Reply:
changed it and added it to `extensions` instead